src/utils/_variables.scss

```
@@ -0,0 +1,5 @@
+$color-header-bg: #fff;
```

It is a good practive to name color variables more descriptively like so:
```suggestion
$color-white: #fff;
```

It is then much easier to visualize the UI when reading CSS :)